content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md

```diff
@@ -44,7 +44,7 @@ For more details, please read the GCR
 [documentation](https://cloud.google.com/container-registry/docs/).
 
 ```shell
-docker build -t my-kube-scheduler:1.0 .
+docker build -t gcr.io/my-gcp-project/my-kube-scheduler:1.0 .
```

Comment:
Not sure, if you are pushing it to google cloud container registry. If you are pushing to google container registry the build command seems correct.

Reply:
The next line in the document is `$ gcloud docker -- push gcr.io/my-gcp-project/my-kube-scheduler:1.0`

It will result in the following error:

```
The push refers to repository [gcr.io/my-gcp-project/my-kube-scheduler]
An image does not exist locally with the tag: gcr.io/my-gcp-project/my-kube-scheduler
```